project/build.scala

```diff
@@ -48,7 +48,12 @@ object MarathonBuild extends Build {
         libraryDependencies ++= Dependencies.root,
         parallelExecution in Test := false,
         sourceGenerators in Compile <+= buildInfo,
-        buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion),
+        buildInfoKeys := Seq[BuildInfoKey](
+          name, version, scalaVersion,
+          BuildInfoKey.action("buildref") {
+            val suffix = Process("git diff --shortstat").lines.headOption.map(_ => "-dev").getOrElse("")
+            Process("git rev-parse HEAD").lines.headOption.getOrElse("unknown") + suffix
```

Comment:
This will fail if git is not in path. Please call in a `Try`
```
Try(Process("git rev-parse HEAD").lineStream).toOption.flatMap(_.headOption).getOrElse("unknown") + suffix
```
The same for applies for shortstat.